### DISCUSSION TITLE
One central repo versus forks

### DISCUSSION THREAD

**Discussion by coderextreme** [christoph-v/spark#12] [Q&A] [answered]
Would you prefer a single repo or do pull requests from forks?

thanks!

**Comment by christoph-v** [comment, marked as answer]
It depends. There will be GitHub users, who will be allowed to work directly with the spark repo as collaborators (e.g. you, @coderextreme ).
On the other hand, each GitHub user will have the possibility to fork and create pull requests (PRs).

If anybody wants to become collaborator, just ask.

**Comment by coderextreme** [comment]
I would like to be a collaborator, if not already to check in Grasshopper
Academy pre-alpha code into 3rd-party-input.

Thanks!   I will probably get to it in the next day or so.

Have you looked into Next.js?

John

**Reply by christoph-v** [reply]
I added you as collaborator some time ago, when it was about to add something to the Wiki.

**Reply by christoph-v** [reply]
[...]Have you looked into Next.js?[...]
Not yet. It's bed time in Austria now :-) Have to go to work tomorrow (although in Home Office, it's challenging anyway).
